src/js/components/DataTable/doc.js

```diff
@@ -66,6 +66,13 @@ export const doc = DataTable => {
       be combined with properties that expect all data to be present in the
       browser, such as columns.search, sortable, groupBy, or columns.aggregate.`,
     ),
+    onClickRow: PropTypes.func.description(
```

Comment:
We should also add documentation to the theme-props on doc.js.
`props.theme.table.row.hover.background` 
`props.theme.table.row.hover.color`

Reply:
good catch!